docs/core/tutorials/using-on-mac-vs-full-solution.md

```diff
@@ -144,17 +152,17 @@ Unit tests provide automated software testing during your development and publis
 
    The `CountInstancesCorrectly` checks that the `GetWordCount` method counts correctly. The `InlineData` provides a count, a search word, and an input string to check. The test method runs once for each line of data. Note once again that you're asserting a failure first by using `Assert.NotEqual`, even when you know that the counts in the data are correct and that the values will match the counts returned by the `GetWordCount` method. Performing the step of failing the test on purpose might seem like a waste of time at first, but checking the logic of the test by failing it first is an important check on the logic of your tests. Eventually, you'll probably come across a test method that passes when you expect it to fail and find a bug in the logic of the test. It's worth the effort to take this step every time you create a test method.
```

Comment:
The ultimate solution is provided for download, and so the sample shown in the doc shows the final code (after `NotEqual` is used+changed), which doesn't jive with the text. I propose we just change the sample from a link to the sample here to a code block in the document that shows `NotEqual` to match the text.